### PR TITLE
docs: introduce app secret

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add "App Secret" tip to "Next.js" and "Traditional Web", also, fix code on `requester` for "Traditional Web".

## Testing

截屏2022-08-03 下午3 42 48

### CODE REVIEW

File: docs/docs/recipes/integrate-logto/fragments/_get-app-secret.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import appSecret from '../assets/app-secret.png';
+
+:::tip
+
+<p>You can find and copy "App Secret" in Application details page of Admin Console:</p>
```
Comment (charIeszhao) [inline docs/docs/recipes/integrate-logto/fragments/_get-app-secret.mdx:5]:
```suggestion
<p>You can find and copy "App Secret" in application details page of Admin Console:</p>
```

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 2]
```diff
@@ -56,6 +59,7 @@ import LogtoClient from '@logto/next';
 
 export const logtoClient = new LogtoClient({
   appId: '<your-application-id>',
+  appSecret: '<your-app-secret-copied-from-console>',
```
Comment (charIeszhao) [inline docs/docs/recipes/integrate-logto/next-js.mdx:62]: The `appSecret` is an optional config, right? This PR makes it look like it's required

Comment (wangsijie) [reply docs/docs/recipes/integrate-logto/next-js.mdx:62]: For traditional web app, it IS required, without `appSecret`, the grant will fail.

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 3]
```diff
@@ -126,11 +128,22 @@ const { fromUint8Array } = require('js-base64');
...
 
-const requester = createRequester(fetch);
+const requester = (input, init) => {
+  const { appId, appSecret } = config;
+
+  return fetch(input, {
+    ...init,
+    headers: {
+      ...init?.headers,
+      authorization: `basic ${Buffer.from(`${appId}:${appSecret}`, 'utf8').toString('base64')}`,
```
Comment (gao-sun) [inline docs/docs/recipes/integrate-logto/traditional.mdx:143]: should be the code below?

```suggestion
      authorization: `Basic ${Buffer.from(`${appId}:${appSecret}`, 'utf8').toString('base64')}`,
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/traditional.mdx [hunk 4]
```diff
@@ -126,11 +129,22 @@ const { fromUint8Array } = require('js-base64');
...
 
-const requester = createRequester(fetch);
+const requester = (input, init) => {
+  const { appId, appSecret } = config;
+
+  return fetch(input, {
+    ...init,
+    headers: {
+      ...init?.headers,
+      authorization: `basic ${Buffer.from(`${appId}:${appSecret}`, 'utf8').toString('base64')}`,
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/traditional.mdx:144]: ditto if it makes sense

```suggestion
      authorization: `Basic ${Buffer.from(`${appId}:${appSecret}`, 'utf8').toString('base64')}`,
```

**Review by gao-sun — APPROVED** [review]
good - so we are going to allow machine to machine app based on this technique?

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/traditional.mdx (hunk 4 above)
Comment (wangsijie) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/traditional.mdx:144]: We are almost there.

**Pull request merged**